doc/usage/unitpackage.md

```diff
@@ -254,21 +254,21 @@ entry.package
 
 One resource is named according to the CSV filename. The units provided in that resource are describing the data within that CSV.
 
-The second resource is called `echemdb`. It is created upon loading a datapackage with the `unitpackage` module and stores the data of the CSV as a pandas dataframe. The dataframe is directly accessible from the entry and shows the data from the `echemdb` resource. Upon loading the data, both the numbers and units in the CSV and pandas dataframe are identical.
+The second resource is called `unitpackage_resource`. It is created upon loading a datapackage with the `unitpackage` module and stores the data of the CSV as a pandas dataframe. The dataframe is directly accessible from the entry and shows the data from the `unitpackage_resource` resource. Upon loading the data, both the numbers and units in the CSV and pandas dataframe are identical.
```

Comment:
The text "`unitpackage_resource` resource" sounds odd, but I think it makes sense in the code to denote the resource with that name.